stories/index.js

```diff
@@ -25,33 +27,27 @@ storiesOf('Range Components/SingleRange', module)
   .addDecorator(withKnobs)
   .add('Basic', () => ({
     components: { BaseSingleRange },
-    template: '<base-single-range/>',
+    template: '<base-single-range :subProps="{ showFilter: false }"/>',
   }))
   .add('with title', () => ({
     components: { BaseSingleRange },
-    methods: { text:(x,y) => text(x,y)
-    },
-    template: '<base-single-range :subProps="{ title: text(\'title\',\' Books Filter \')}"/>',
+    mixins:[addons],
```

Comment:
Cool, but instead of adding it here in each component define global mixings in below file to avoid code duplication.
https://github.com/appbaseio/vue-playground/blob/master/.storybook/config.js

Reply:
I tried doing that but it was not working let me give it a try again.

Reply:
@bietkul updated 😄